dev/Materials/Acrylic/AcrylicBrush.cpp

```diff
@@ -595,6 +595,15 @@ winrt::CompositionEffectBrush AcrylicBrush::CreateAcrylicBrushWorker(
         }
     }
 
+#ifndef BUILD_WINDOWS
+    winrt::CompositionObject compositionObject{ acrylicBrush.try_as<winrt::CompositionObject>() };
+    winrt::CompositionPropertySet compositionPropertySet = compositionObject.Properties();
+
+    // Property sets don't support strings as property values, so store our tag as a scalar value.
+    static winrt::hstring brushIdentifier = winrt::hstring(L"ShouldRenderAsFallbackInIslands");
```

Comment:
Re "static winrt::hstring brushIdentifier = winrt::hstring(L"ShouldRenderAsFallbackInIslands");"

Using LuminosityColorColor as an example, I think this can just be: L"ShouldRenderAsFallbackInIslands"sv

This, combined with my other comment above, might mean these lines can collapse to:
```suggestion
    acrylicBrush.Properties().InsertScalar(L"ShouldRenderAsFallbackInIslands", 1.0f);
```